.github/workflows/dependabot_update_rbis.yml

```diff
@@ -0,0 +1,71 @@
+name: Update gem RBIs on Dependabot PRs
```

Comment:
Once we are happy with this, it can be moved to another repo for sharing.

Reply:
Ruby-lsp is already public. Why not have this in tapioca from the start?

Reply:
Just so that if we need to tweak anything, then it's all in the one place, rather than having to make changes across two repos.